jfix-stdlib-concurrency/src/main/java/ru/fix/stdlib/concurrency/threads/ReschedulableScheduler.java

```diff
@@ -17,12 +22,18 @@
 
     private final ScheduledExecutorService executorService;
 
+    private final Set<SelfSchedulableTaskWrapper> activeTasks;
```

Comment:
not thread safe!

Reply:
change to thread-safe set from ConcurrentHashMap